openapi/api_api.go

```diff
@@ -172,7 +177,7 @@ func StartSIOPSameDevice(c *gin.Context) {
 
 // VerifierAPIAuthenticationResponse - Stores the credential for the given session
 func VerifierAPIAuthenticationResponse(c *gin.Context) {
-	state, stateExists := c.GetQuery("state")
+	state, stateExists := c.GetPostForm("state")
```

Comment:
Wouldn't this be a breaking change?

Reply:
Yep. Thats why I used the chance to do it know, while we do a major upgrade anyways. Do you think its better to allow both for the time beeing?

Reply:
done